app/src/main/java/fr/gaulupeau/apps/Poche/service/BGService.java

```diff
@@ -58,54 +58,40 @@ public BGService() {
     protected void onHandleIntent(Intent intent) {
         Log.d(TAG, "onHandleIntent() started");
 
-        // this seems to make UI more responsive right after a call to the service.
-        // well, this seemed to help better before OperationsHelper introduction
         android.os.Process.setThreadPriority(android.os.Process.THREAD_PRIORITY_BACKGROUND);
 
         ActionRequest actionRequest = ActionRequest.fromIntent(intent);
         ActionResult result = null;
 
         switch(actionRequest.getAction()) {
-            case SyncQueue: {
-                SyncQueueStartedEvent startEvent = new SyncQueueStartedEvent(actionRequest);
-                postStickyEvent(startEvent);
-                try {
-                    result = syncOfflineQueue(actionRequest);
-                } finally {
-                    removeStickyEvent(startEvent);
-                    if(result == null) result = new ActionResult(ActionResult.ErrorType.Unknown);
-                    postEvent(new SyncQueueFinishedEvent(actionRequest, result));
+            case Archive:
+            case Unarchive:
+            case Favorite:
+            case Unfavorite:
+            case Delete:
+            case AddLink:
```

Comment:
I considered moving these into the `default` case (another check is done inside of `serveSimpleRequest()` anyway), but I don't see improvements in such move. I thought that it will be easier to make a decision when there will be need to change this part again.